extensions/Khronos/KHR_lights/schema/light.schema.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "light",
+    "type": "object",
+    "description": "An ambient, hemisphere, directional, point, or spot light.",
+    "allOf" : [ { "$ref" : "../../../../specification/2.0/schema/glTFChildOfRootProperty.schema.json" } ],
+    "properties": {
+        "color": {
+            "type": "array",
+            "subtype": "number",
+            "description": "Color of the light source.",
+            "default": [1, 1, 1]
+        },
+        "intensity": {
+            "type": "number",
+            "description": "Intensity of the light source in lumens.",
```

Comment:
It should probably be clarified that the intensity either has units of candela or lux, depending on the light type.